src/lib/server/blog/posts.ts

```diff
@@ -84,7 +84,7 @@ function cacheBlogPosts(output: any): BlogPost[] {
     const officer = getOfficerByGhUsername(author.login);
     const authorUrl: string = author.url;
     const displayname: string = officer?.fullName ?? author.login;
-    const picture: string = officer?.picture ?? author.avatarUrl;
+    const picture: string = author.avatarUrl ?? officer?.picture;
```

Comment:
The `officer?.picture` contains just the filename (e.g. `ethan-davidson.webp`).
```suggestion
    const picture: string = author.avatarUrl ?? `/assets/authors/${officer?.picture || 'placeholder.webp'}`;
```
Note: I am thinking that once we resolve #546, we can go back to using `officer?.picture` before `author.avatarUrl`.

Reply:
do u want me to commit that suggestion?

Reply:
> The `officer?.picture` contains just the filename (e.g. `ethan-davidson.webp`).
> 
> Note: I am thinking that once we resolve #546, we can go back to using `officer?.picture` before `author.avatarUrl`.

Actually, since our blog post is posted through Github and we also link our Github profile to the blog author, I think it's better to have the author picture same as Github pfp? thoughts?

Reply:
> Actually, since our blog post is posted through Github and we also link our Github profile to the blog author, I think it's better to have the author picture same as Github pfp? thoughts?

We plan to store the GitHub usernames of most of our current board members in `officers.json`, so the we are technically able to then reference the board member's information. This includes information such as the board member's full name (used in a line of code above) and custom photo instead of their default GitHub profile picture. 

If the blog post author is a board member with a known GitHub username, we should be able to use the board member's headshot.

Reply:
make sense, pushing changes now